imod/mf6/simulation.py

```diff
@@ -1014,6 +1027,9 @@ def _set_exchange_options(self):
                     xt3d=model_1["npf"].get_xt3d_option(),
                     newton=model_1.is_use_newton(),
                 )
+            if isinstance(exchange, GWTGWT):
```

Comment:
This can be changed to:

```suggestion
            elif isinstance(exchange, GWTGWT):
```